getting-started/kubernetes/flannel/flannel.md

```diff
@@ -79,3 +82,6 @@ etcd, complete the following steps.
 
 1. If you wish to enforce application layer policies and secure workload-to-workload
    communications with mutual TLS authentication, continue to [Enable application layer policy]({{site.baseurl}}/security/app-layer-policy) (optional).
+
+{% include geek-details.html details='Policy:Calico,IPAM:Host-local,CNI:Calico,Overlay:VXLAN,Routing:Static,Datastore:etcd' %}   
```

Comment:
Did you intentionally put this second one at the bottom of the flannel page? Seems odd to me.

Reply:
It's meant to be there, but is missing the intro sentence "The geeky details of what you get:".  

I'll fix that up.